### DISCUSSION TITLE
Enforce uniqueness on a column via UI

### DISCUSSION THREAD

**Discussion by jasonho-lynx** [supabase/supabase#1102] [Questions] [answered]
Hello, is it possible to enforce uniqueness on a column via the UI? Or do we have to write the query for that? Not sure if I'm missing some option somewhere.

Thank you!

**Comment by jasonho-lynx** [comment]
Seems to be a no, based on this comment [here](https://github.com/supabase/supabase/discussions/713#discussioncomment-365936), but if anyone knows otherwise, please let me know

**Comment by silentworks** [comment, marked as answer]
This is currently not possible in the **Table editor** in the UI, but you can use the **SQL** section in the UI to write a query.
